src/components/articleTeaser/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react';
+import { chakra, ResponsiveValue, useMultiStyleConfig } from '@chakra-ui/react';
+
+import Stack from '../stack';
+import { Sizes } from '../../themes';
+
+type Props = {
+  title: string;
+  text: string;
+  imageUrl: string;
+  url: string;
+  maxImgW?: ResponsiveValue<Sizes>;
+};
+
+const ArticleTeaser: FC<Props> = ({
+  title,
+  text,
+  imageUrl,
+  url,
+  maxImgW = '4xl',
+}) => {
+  const styles = useMultiStyleConfig('ArticleTeaser');
+
+  return (
+    <article>
+      <a href={url} target="_blank" rel="noopener noreferrer">
+        <Stack
+          direction={{
+            xs: 'column',
+            md: 'row',
+          }}
+          spacing={{ md: 'lg' }}
+          align="center"
+        >
+          <chakra.img maxW={maxImgW} src={imageUrl} />
+          <Stack direction="column" spacing="sm">
+            <chakra.h2 __css={styles.title}>{title}</chakra.h2>
```

Comment:
Not sure, but I think only the title should be the one who linked us. Maybe I'm missing something here.

Reply:
I checked that with Enrique, he said that it's better if it's the entire teaser. Will add that to notes and ticket